prelude/ts/src/sk_node.ts

```diff
@@ -20,7 +21,9 @@ class WrkImpl implements Wrk {
     url: URL,
     options: WorkerOptions | undefined,
   ): Wrk {
-    const filename = "." + url.pathname.substring(process.cwd().length);
+    const cwd = process.cwd();
+    const child = url.pathname.startsWith(cwd + path.sep);
+    const filename = child ? "." + url.pathname.substring(cwd.length) : url;
```

Comment:
Another version:
```js
const filename = path.relative(process.cwd(), url.pathname);
```
I don't know which is better :shrug:

Reply:
I hesitated to do it, I asked myself the same question. It's true that it's more concise.

Reply:
But the if it's relative path it must start with "./"